### PR TITLE
Add target problem and persona to main Company page

### PR DESCRIPTION
**Pull request by tayloramurphy**
Based on our Leadership meeting discussion on the problem and persona.

@meltano/leadership

### CODE REVIEW

**Review by therebbie — COMMENTED** [review]
I like this -- my only change would be that we seek to **be** the foundation, not to become it.  I think there is an important difference in that change because "be" is more active vs. hoping for something when one uses the word "become."

File: src/_company/index.md [hunk 1]
```diff
@@ -34,17 +34,26 @@ To learn more about this mission, check out this [detailed blog post](https://me
 
 ## Vision
 
-Our vision is for Meltano to become **the foundation of every team's ideal data stack**.
+Our vision is for Meltano to become **the foundation of every team's ideal data platform**.
```
[the remark this answers is not in the document]
Comment (tayloramurphy) [reply src/_company/index.md:37]:
```suggestion
Our vision is for Meltano to be **the foundation of every team's ideal data platform**.
```

**Comment by emilylucie** [comment]
thanks @tayloramurphy for doing this. It looks good. I still don't love that we are saying people are trying to build a platform but I will let it go. They just want best-in-class tools and in an ideal world they would work together, but I don't think they think of it as a platform.  They find us because they have individual tools and specifically not a platform- we provide that functionality. I would be more comfortable with stack but I can let it go since it is semantics.

I am okay if we merge. 

I have some additional detail about the person I would still like to flush out but not sure how much competitive detail we want to out out there- for a later discussion. Thinking we need info such as what are their consumption habits and unique identifiers.

File: src/_company/index.md [hunk 2]
```diff
@@ -34,17 +34,26 @@ To learn more about this mission, check out this [detailed blog post](https://me
...
+Our mission and vision are broad. To be able to achieve them, we must find product-market fit with a targeted profile of users so that we can build a sustainable business and continue to grow. 
 
-You can learn more about our strategy on the [Strategic Goals](strategy) page.
+Our target persona are mid- to senior-level Data Engineers who work for companies with 30-200 people and are likely the only data professional in their company.
+
+The problem we are solving for them is that they are in the process of setting up a new data platform that will enable their organization to generate valuable insights, but they are dissatisfied with their previous experience and want a solution that addresses many of the problems they encountered as their previous platform attempted to change and grow. Their previous experience likely include:
```
Comment (tayloramurphy) [inline src/_company/index.md:49]:
```suggestion
The problem we are solving for them is that they are in the process of selecting different tools, what we would call "setting up a new data platform", that will enable their organization to generate valuable insights, but they are dissatisfied with their previous experience and want a solution that addresses many of the problems they encountered as their previous platform attempted to change and grow. Their previous experience likely include:
```

@emilylucie what do you think about a caveat / change like this?

Comment (DouweM) [reply src/_company/index.md:49]: I like "data stack" better than "data platform", as I think data engineers do think of "picking tools" as equivalent to "putting together a data stack"

```suggestion
The problem we are solving for them is that they are in the process of putting together a new data stack that will enable their organization to generate valuable insights, but they are dissatisfied with their previous experience and want a solution that addresses many of the problems they encountered as their previous platform attempted to change and grow. Their previous experience likely include:
```

**Comment by DouweM** [comment]
I'm taking an iteration on this.

**Comment by DouweM** [comment]
To do for me: Update with "Meltano+Singe+dbt+Airflow = Fivetran killer" framing.

**Comment by DouweM** [comment]
Replaced by https://github.com/meltano/internal-general/pull/433.

**Pull request closed without merging**